### PR TITLE
LCRB issuer configurations

### LINKED ISSUE #127: Incorporate configurations for LCRB OrgBook issuer
Migrate and update the LCRB OrgBook issuer configurations from [here](https://github.com/bcgov/jag-lcrb-carla-public/tree/develop/orgbook-issuer-agent).

DTS will be hosting/maintaining the issuer agent/controller for LCRB moving forward.

When done, create and submit PR to remove the component from the original repo as well.

### PR DESCRIPTION
**Pull request by esune**
Added OpenShift templates and configurations to support the LCRB issuer for OrgBook.

Controller code is built from [aries-vcr-issuer-controller](https://github.com/bcgov/aries-vcr-issuer-controller) and configuration files are overlayed at runtime using a configmap.

Agent and wallet configurations were copied from the `lsbc` deployments and tweaked accordingly.

I am unsure whether `ledgers.yaml` or `ledgers-bcreg.yaml` should be used for the agent - they seem to have similar configurations as far as I can tell. Additionally, the `dev` configurations point at BCovrin Test rather than Dev in those files.
@WadeBarnes do you have input about this?

I have NOT yet deployed anything as some coordination with LCRB to migrate their data to the namespace is required.

Resolves #127

### CODE REVIEW

**Comment by WadeBarnes** [comment]
@esune, use the `ledgers-bcreg.yaml` files.

File: openshift/templates/orgbook-issuer-controller/orgbook-issuer-controller-build.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  - name: DOCKER_FILE_PATH
+    displayName: Docker File Path
+    description: The path to the docker file.
+    required: true
+    value: Dockerfile
+  - name: SOURCE_IMAGE_NAME
+    displayName: Source Image Name
+    description: The name of the Docker image to be used for the build. Overrides the FROM instruction in the Dockerfile.
+    required: false
+    value: artifacts.developer.gov.bc.ca/docker-remote/bcgovimages/von-image:py36-1.16-0
```
Comment (WadeBarnes) [inline openshift/templates/orgbook-issuer-controller/orgbook-issuer-controller-build.yaml:105]: Does the controller need to be based on `von-image` or can we use a generic python container like we're doing now with the OrgBook API?

Comment (esune) [reply openshift/templates/orgbook-issuer-controller/orgbook-issuer-controller-build.yaml:105]: We might be able to switch to a generic Python image, I need to double check. I believe looked into this a while back and there was something that prevented us from uplifting it, but something prevented us from moving forward. Will review and get back to you as I stand-up the service.

**Comment by esune** [comment]
> So we're hosting their controller for them too. Interesting. One question/comment on the controller below.

Yes, this is consistent with the pattern we have established for other OrgBook issuers (BuyBC) using the Aries VCR Issuer Controller component. I think in this scenario, until we switch to the Aries VCR Issuer Agency, we will have to host the controllers as well since they are tightly coupled with the agent and credential issuance process, and clients seem to have issues managing them either way.

I am open to exploring a different option if it seems more appropriate though.

**Comment by esune** [comment]
@WadeBarnes this is now ready to go, services have been deployed in `dev/test/prod`.

A couple of highlights of the changes:
* the `backup` component was updated to include settings for ALL environments, and only require commenting-out non-applicable settings when deployed/updated (currently the IDIM agent settings)
* some tweaks to `orgbook-issuer-controller`, including using vanilla Python base image
* added new `orgbook-issuer-agent` build/deployment configuration as extra plugins are required for OrgBook issuers.
* consolidated ledger configurations for OrgBook issuers to be in one file per environment, rather than having multiple files.

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**